packages/oui-checkbox/_mixins.less

```diff
@@ -221,7 +221,7 @@
     }
 
     .@{checkbox-selector}__input:not(:disabled) + .@{checkbox-selector}__label-container:hover,
-    .@{checkbox-selector}__input:checked:not(:disabled) + .@{checkbox-selector}__label-container:hover {
+      .@{checkbox-selector}__input:checked:not(:disabled) + .@{checkbox-selector}__label-container:hover {
```

Comment:
Saw together, it's a problem of `postcss-less` and `stylelint`. You can get more information here: https://github.com/stylelint/stylelint/issues/1741